app/kubernetes-ingress-controller/1.0.x/deployment/k4k8s-enterprise.md

```diff
@@ -50,7 +50,7 @@ Execute the following to install Kong for Kubernetes Enterprise using YAML
 manifests:
 
 ```bash
-$ kubectl apply -f https://bit.ly/k4k8s-enterprise-install
+$ kubectl apply -f https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/v{{ page.version }}/deploy/single/all-in-one-dbless.yaml-enterprise-install
```

Comment:
```suggestion
$ kubectl apply -f https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/v{{ page.version }}/deploy/single/all-in-one-dbless-k4k8s-enterprise.yaml
```
Looks like a search-and-replace goof.

I'm putting in `all-in-one-dbless-k4k8s-enterprise.yaml` in all of the wrongly replaced ones as suggestions, might want to double-check if any of them are supposed to be `all-in-one-postgres-enterprise.yaml`.